vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -148,6 +148,15 @@
 .dropdown-container .dropdown > .dropdown-item.selected,
 .dropdown-container .dropdown > .dropdown-item.highlighted {

```

Comment:
If the change below is applied (using opacity), it is required to disable the hover and active styling:
```
.dropdown-container .dropdown > .dropdown-item:not(.disabled):hover,
.dropdown-container .dropdown > .dropdown-item:not(.disabled):active,
.dropdown-container .dropdown > .dropdown-item.selected,
.dropdown-container .dropdown > .dropdown-item:not(.disabled).highlighted {
    background-color: $soft-blue;
    font-weight: 700;
}
```